docs/book/user-guide/starter-guide/using-project-templates.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Rocketstart your ZenML journey!
+---
+
+# ZenML project templates
```

Comment:
Considering the other titles in the starter guide, a title like `Project Templates` might be a bit more fitting here imo. (this would require a change in the `toc.md` as well.)

Reply:
Renamed :+1